lib/gh-koenig/addon/components/gh-koenig.js

```diff
@@ -90,8 +91,18 @@ export default Component.extend({
         }
         let editor = this.get('editor');
 
-        editor.didRender(() => {
+        editor.willRender(() => {
+            if (!run.currentRunLoop) {
+                run.begin();
+                this.set('inRunLoop', true);
```

Comment:
For purely internal properties there's no need for get/set - you only need those when you want KVO bindings (and the additional overhead that goes with them). Also, if the property is not exposed to the outside world the convention is to prefix with an underscore. Eg.

```
if (this._inRunLoop) {
    run.end();
    this._inRunLoop = false;
}
```

It would also be useful to add a comment here explaining why manual runloop tracking is needed - I had to look it up in the [ember-mobiledoc-editor](https://github.com/bustlelabs/ember-mobiledoc-editor/blob/master/addon/components/mobiledoc-editor/component.js#L230-L235) code